### PR TITLE
Added initial image listing endpoint

### LINKED ISSUE #44: Add support for listing images

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #44

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -1234,6 +1234,29 @@ paths:
...
+        - MediaLibrary
+      responses:
+        "200":
+          description: List of images
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Image"
```
Comment (anttileppa) [inline swagger.yaml]:
```suggestion
                  $ref: "#/components/schemas/MediaFile"
```

File: swagger.yaml [hunk 2]
```diff
@@ -1234,6 +1234,29 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+
+  /v1/images:
```
Comment (anttileppa) [inline swagger.yaml]: images -> mediaFiles

File: swagger.yaml [hunk 3]
```diff
@@ -1234,6 +1234,29 @@ paths:
...
+    get:
+      operationId: listImages
+      summary: Lists images
+      description: Lists images from S3 bucket
+      tags:
```
Comment (anttileppa) [inline swagger.yaml:1243]: Add mandatory parameter "path"

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (villejuutila) [reply swagger.yaml]: Resolved.

File: swagger.yaml (hunk 2 above)
Comment (villejuutila) [reply swagger.yaml]: Resolved.

File: swagger.yaml (hunk 3 above)
Comment (villejuutila) [reply swagger.yaml:1243]: Resolved.

**Review by anttileppa — APPROVED** [review]

**Pull request merged**